exercises/06-deploy-to-cf.md

```diff
@@ -1,29 +1,41 @@
 # Deploy your application to Cloud Foundry
+
 Now, that you made some progress locally, let's make this application available in the cloud.
 
 ## Login to Cloud Foundry
-Login to Cloud Foundry using the following command on the command line (if you haven't logged in before):
+
+You need to login to Cloud Foundry in SAP Business Technology Platform using the cf CLI.First, you need to set an API endpoint. The exact URL of this API endpoint depends on the region your _subaccount_ is in. Open the [SAP Business Technology Platform Cockpit](https://account.hana.ondemand.com/) and navigate to the subaccount you are planning to deploy your application. Click on _Overview_ on the left and you can see the URL of the API endpoint.
+
+![SAP BTP Cockpit Overview](https://sap.github.io/cloud-sdk/img/subaccount_api_endpoint.png)
+
+Copy the URL and paste it into the following command in your command line (if you haven't logged in before):
 
 ```sh
-cf login -a https://api.cf.eu10.hana.ondemand.com
+cf login -a https://api.cf.<region>.hana.ondemand.com
```

Comment:
[req] use the `YOUR_` format as in the style guide

Reply:
follow style guide here too